packages/mcp-client/src/McpClient.ts

```diff
@@ -23,7 +29,7 @@ export interface ChatCompletionInputMessageTool extends ChatCompletionInputMessa
 
 export class McpClient {
 	protected client: InferenceClient;
-	protected provider: InferenceProviderOrPolicy | undefined;
+	protected provider: InferenceProvider | undefined;
```

Comment:
you'll need to merge main as this was changed (very slightly) on main

Reply:
(meaning `InferenceProviderOrPolicy` is the new type – though i agree it's a mouthful) ie. you can just revert those changes here and below

Reply:
basically what this does is we now accept some "virtual" providers like `"auto"` (which points to your preferred provider in your HF settings that serve the model in question)

cc @Wauplin @hanouticelina from our client SDK team